src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -259,6 +259,10 @@ def load_command_table(self, _):
         g.custom_command('add', 'add_github_actions')
         g.custom_command('remove', 'remove_github_actions')
 
+    with self.command_group('functionapp deployment github-actions') as g:
+        g.custom_command('add', 'add_functionapp_github_actions')
+        g.custom_command('remove', 'remove_functionapp_github_actions')
```

Comment:
Do we need to consider adding `confirmation=True` for the remove command?

Reply:
We do not do it for similar command `webapp deployment github-actions`